### ISSUE TITLE
[RFC] Allow option to automatically add all team members as owners

### ISSUE THREAD

**Issue by aikocc** [Voxel-Fox-Ltd/VoxelBotUtils#78]
Add a config options that's either `True` or `False` that when set to `True` will if the bot's application is a team application it will automatically add all team members as owners.

**Comment by 4Kaylum** [comment]
Adding another config option for it seems unnecessary - how about an empty `owner_ids` list evaluating to the team members?

**Comment by aikocc** [comment]
Yeah, like if nothing is specified then it will get the application owner or team members.

**Comment by 4Kaylum** [comment]
I'm not adverse to this at all - seems fairly reasonable. It's actually some behaviour I broke from default D.py, so yeah. Adding it back would be good.

**Comment by 4Kaylum** [comment]
The only issue I have is that it's no longer explicit who the owners are marked as, but I guess that's fine, really.

**Comment by GlitchMasta47** [comment]
I could see the usage for it as a separate config option if you wanted to have application owners added, and an explicit list of other owners in case they want to add an owner without giving them full access to the bot application or something similar. To be fair, it's not like they don't have essentially full access with owner anyways since they can eval whatever they want, but it's still an idea.
